frontend/dashboard/src/keycloak.ts

```diff
@@ -0,0 +1,12 @@
+import Keycloak from "keycloak-js";
+
+//Keycloak init options
+const initOptions = {
+  url: "https://authn.diamond.ac.uk/",
+  realm: "master",
+  clientId: "iat69393",
```

Comment:
Shouldn't we have a generic `clientId` ?

Reply:
Yeah, should probably add a workflows clientId for testing purposes